lib/routes/yilin/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Data, DataItem, Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/',
+    categories: ['reading'],
+    example: '/yilin',
+    radar: [
+        {
+            source: ['www.yilinzazhi.com'],
+            target: '/',
+        },
+    ],
+    name: '文章列表',
+    maintainers: ['g0ngjie'],
+    handler,
+    url: 'www.yilinzazhi.com',
+};
+
+async function handler(): Promise<Data> {
+    const baseUrl = 'https://www.yilinzazhi.com/';
+    const response = await got(baseUrl);
+    const $ = load(response.data, { xmlMode: true });
```

Comment:
```suggestion
    const $ = load(response.data);
```
`xmlMode` is not needed for HTML content.